graph/telemetry/istio/appender/kiali_annotation.go

```diff
@@ -0,0 +1,69 @@
+package appender
+
+import (
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/graph"
+	"github.com/kiali/kiali/models/annotation"
+)
+
+const AnnotationAppenderName = "kialiAnnotation"
+
+// Name: annotation
+type AnnotationAppender struct{}
```

Comment:
This would change to `HealthConfigAppender`.  Also, the appenders typically have a fairly robust comment above type decl.

Also, before a merge we'd need test code for this appender.